### ISSUE TITLE
Verifying the Firewall VM

### ISSUE THREAD

**Issue by dkindlund** [dkindlund/honeyclient#90]
Hi Darien

I managed to chown the firewall-2 directory to be devinder:admin.

The Firewall VM starts and I had logged in as "roo" using password "password", but I cant see anything when I type: 

{{{
$ ps ax | grep startFW | grep -v grep
}}}

--Devinder

**Comment by dkindlund** [comment]
Author: kindlund
Hi Devinder,

That is odd.

Okay, can you post your output of just "ps ax" ?  Once I have that, I can try to figure out why it's not there.

Keep in mind that it may take a little time for the firewall to fully start up.  Once you power this VM on, it may take about 3 minutes for the perl script to run.  However, if it still is not running, then something may be wrong when it booted up.

Regards,

-- Darien

**Comment by dkindlund** [comment]
Author: Devinder devinbhullar@gmail.com
Hi Darien 

I has uploaded the screen captures from the firewall-2 VM 

$ps ax

Regards,
Devinder

**Comment by dkindlund** [comment]
Author: kindlund
Hi Devinder,

Okay, I think I understand the problem.  It seems that the perl code inside the Firewall VM did not automatically start.  This is usually the result of some service significantly failing at startup.

As you requested, we can try to troubleshoot this specific problem over IM.  I'll be sure to get online when I can.  In the meantime, you can reboot the Firewall VM and see if you notice any specific failures upon startup.  When the VM is fully booted up, you can use SHIFT+PageUp and SHIFT+PageDn keys to scroll through the screen buffer, to take screenshots of any failed services.

Hope this helps,

-- Darien

**Comment by dkindlund** [comment]
Author: kindlund
Okay, after chatting directly with Devinder, we narrowed down the problems to be:

1) Devinder did not follow Step 8 in this section of the UserGuide:
http://www.honeyclient.org/trac/wiki/UserGuide#ConfigureVMwareServer

As such, the dhcpd server running on vmnet1 was not properly configured.

2) Additionally, there was another VM running in additional to the "Firewall VM" that had already claimed the IP address of 10.0.0.254.  Had the dhcpd server been configured properly, this IP address would have never been issued.

Closing this ticket.
